src/shared-components/carousel/test.tsx

```diff
@@ -1,30 +1,32 @@
 import React from 'react';
-import { renderer } from 'src/tests/reactTestRendererHelpers';
+import { render } from 'src/tests/reactTestingLibraryHelpers';
 
 import { Carousel } from './index';
 
-const cards = [<Carousel.Card key="card1">Card 1</Carousel.Card>];
+const cards = [
+  <Carousel.Card key="card1">Card 1</Carousel.Card>,
+  <Carousel.Card key="card2">Card 2</Carousel.Card>,
+  <Carousel.Card key="card3">Card 3</Carousel.Card>,
+];
 
 describe('<Carousel />', () => {
   describe('UI snapshots', () => {
     it('renders with props', () => {
-      const tree = renderer
-        .create(<Carousel numCardsVisible={1}>{cards}</Carousel>)
-        .toJSON();
+      const { container } = render(
```

Comment:
TODO: Make the assertions in this test more meaningful. The snapshot doesn't really do much.